docs/tutorial/multithreading.md

```diff
@@ -42,7 +42,7 @@ safe.
 The only way to load a native module safely for now, is to make sure the app
 loads no native modules after the Web Workers get started.
 
-```js @ts-expect-error=[1]
```

Comment:
This change seems unrelated to the others. Is this required?

Reply:
I wasn't able to commit because of some pre-commit hook that was triggered on that. I can remove that change and ignore pre-commit hook error probably.